test/asset_manager/asset_manager_pact_test.rb

```diff
@@ -0,0 +1,292 @@
+            headers: AssetManagerPactHelper::JSON_CONTENT_TYPE,
+          )
+
+        api_client.whitehall_asset(legacy_url_path)
+      end
+    end
+
+    describe "#update_asset" do
+      it "updates a whitehall asset" do
+        asset_manager
+          .given("a whitehall asset exists with legacy url path #{legacy_url_path} and id #{content_id}")
+          .upon_receiving("an update asset request")
+          .with(
+            method: :put,
+            path: "/assets/#{content_id}",
+            body: a_multipart_request_body,
+            headers: AssetManagerPactHelper::MULTIPART_HEADERS,
+          )
+          .will_respond_with(
+            status: 200,
+            body: created_asset_body(
+              id: "http://example.org/assets/#{content_id}",
+              file_url: "http://static.dev.gov.uk#{legacy_url_path}",
+            ),
+            headers: AssetManagerPactHelper::JSON_CONTENT_TYPE,
+          )
+
+        api_client.update_asset(content_id, file: file_fixture)
+      end
+    end
+
+    describe "#delete_asset" do
+      it "deletes a whitehall asset" do
+        asset_manager
+          .given("a whitehall asset exists with legacy url path #{legacy_url_path} and id #{content_id}")
+          .upon_receiving("a delete asset request")
+          .with(
+            method: :delete,
+            path: "/assets/#{content_id}",
+          )
+          .will_respond_with(
+            status: 200,
+            body: existing_asset_response_body.merge({ "deleted" => true }),
+            headers: AssetManagerPactHelper::JSON_CONTENT_TYPE,
+          )
+
+        api_client.delete_asset(content_id)
+      end
+    end
+
+    describe "#restore_asset" do
+      it "restores a soft deleted whitehall asset" do
+        asset_manager
+          .given("a soft deleted whitehall asset exists with legacy url path #{legacy_url_path} and id #{content_id}")
+          .upon_receiving("a restore asset request")
+          .with(
+            method: :post,
+            path: "/assets/#{content_id}/restore",
+          )
+          .will_respond_with(
+            status: 200,
+            body: existing_asset_response_body,
+            headers: AssetManagerPactHelper::JSON_CONTENT_TYPE,
+          )
+
+        api_client.restore_asset(content_id)
+      end
+    end
+  end
+
+private
+
+  def existing_asset_body(file_url:)
+    {
+      "id" => "http://example.org/assets/4dca570c2975bc0d6d437491",
+      "name" => "asset.png",
+      "content_type" => "image/png",
+      "size" => 57_705,
+      "state" => "uploaded",
+      "file_url" => file_url,
+      "draft" => false,
+      "deleted" => false,
+    }
+  end
+
+  def created_asset_body(id:, file_url:)
+    { "id" => id,
+      "file_url" => file_url,
+      "name" => "hello.txt",
+      "content_type" => "text/plain; charset=utf-8",
+      "size" => 14,
+      "state" => "unscanned",
+      "draft" => false,
+      "deleted" => false }
```

Comment:
```suggestion
    {
      "id" => id,
      "file_url" => file_url,
      "name" => "hello.txt",
      "content_type" => "text/plain; charset=utf-8",
      "size" => 14,
      "state" => "unscanned",
      "draft" => false,
      "deleted" => false
    }
```
Could the curly braces be moved onto new lines, to keep them consistent with the `existing_asset_body` method?